coral-common/src/main/java/com/linkedin/coral/common/functions/FunctionReturnTypes.java

```diff
@@ -45,6 +45,7 @@ private FunctionReturnTypes() {
   public static final SqlReturnTypeInference BINARY = ReturnTypes.explicit(SqlTypeName.BINARY);
   public static final SqlReturnTypeInference BIGINT = ReturnTypes.explicit(SqlTypeName.BIGINT);
   public static final SqlReturnTypeInference SMALLINT = ReturnTypes.explicit(SqlTypeName.SMALLINT);
+  public static final SqlReturnTypeInference INTEGER = ReturnTypes.explicit(SqlTypeName.INTEGER);
```

Comment:
Looks like it's not used?

Reply:
Thanks for pointing it out. I ended up not using it so removing it for now.